### PR TITLE
Add CONTRIBUTING.md, mention mojmap being ARR

### LINKED ISSUE #1209: Conventions.md file is obsolete
The major issue here is that the file doesn't take into account Mojang official mappings. It should indicate the legal issues about these mappings, and the fact that we shouldn't look at them before we make a pull request.

Additionally, as seen with #1189 , the conventions file should make it clear that we preferably use hardcoded identifiers instead of default translations for objects like blocks, items, entities, biomes, etc, when naming classes.

### LINKED ISSUE #1221: Move "Contributions" in readme to "Contributing.md"
A `contributing.md` is detected by github so that all pr senders can see this contribution guidelines, so as to prevent what happened in #1218 from happening again.

### PR DESCRIPTION
**Pull request by Juuxel**
Closes #1221. Created CONTRIBUTING.md file that is basically a copy-paste from the README. **TODO:** Should I remove the contributing part from the README and link to the new file?

Closes #1209. CONVENTIONS.md now mentions that names from mojmap should not be used.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing
+
+Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP or Mojang's obfuscation maps)
+is **completely forbidden** without explicit permission from the owners of said mappings. Discussing the naming approaches used in said projects
+is also not welcome - you have been warned. However, it is a good idea to consult name changes with other people - use pull requests or our community spaces to ask questions!
```
Comment (liach) [inline CONTRIBUTING.md:6]: Naming approach should be fine, but you cannot use their product as an evidence or a premise of your name

Comment (Juuxel) [reply CONTRIBUTING.md:6]: That part is just straight from the README anyways, but I can take that out.

**Review by YanisBft — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (liach) [reply CONTRIBUTING.md:6]: That's good then.

**Review by liach — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ To obtain a deobfuscated Minecraft jar, [`./gradlew mapNamedJar`](#mapNamedJar)
 
 ## Contributing
 
-Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP) is **completely forbidden** without explicit permission from the 
+Please remember that copying and pasting mappings from alternate projects under more restrictive licenses (such as MCP or Mojang's obfuscation maps) is **completely forbidden** without explicit permission from the
```
Comment (modmuss50) [inline README.md]: Should this be more clear that even using their names for insprisation is still forbidden?

Comment (Runemoro) [reply README.md]: > without permission from the owners  of said mappings

Permission to use the names isn't enough. For example, if there was some MIT-licensed mapping project, we'd have permission to use the names, but not distribute under CC0. This sentence should be changed to

> without permission from the owners of said mappings **to distribute the names under the CC0 license**

Comment (Juuxel) [reply README.md]: I added a thing about inspiration and the fix about CC0 in d02a054.

**Review by Runemoro — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**